### ISSUE TITLE
PAF best graph ValueError if the weight of an edge is nan

### ISSUE THREAD

**Issue by mynschen** [DeepLabCut/DeepLabCut#1351]
When evaluating a trained network, the following error occurs at the step of generating best skeleton:
![image](https://user-images.githubusercontent.com/53803897/123306994-8c22eb80-d554-11eb-9dc3-51bd4261d7fd.png)

The problem arises if the weight of an edge is nan. This can be fixed by the following modification to deeplabcut/pose_estimation_tensorflow/lib/crossvalutils.py :
![image](https://user-images.githubusercontent.com/53803897/123307019-9349f980-d554-11eb-8cd9-1f1c26f78085.png)
Basically the idea is to remove edges with weight of nan from the order during sorting.
Hope this would be informative for those trying to familiar themselves with the code using small amount of training and testing data. Ultimately, this should not be a problem if adequate training and testing data were used.

**Desktop (please complete the following information about your system):**
 - OS: Ubuntu 20.04
 - DeepLabCut Version: 2.2rc2
 - DeepLabCut mode: multi-animal tracking
 - GPU RTX2070

**Comment by jeylau** [comment]
Hey @mynschen, we had recently stumbled upon that bug too; it is fixed with #1304 ☺️
